client/src/main/scala/io/delta/sharing/client/model.scala

```diff
@@ -113,6 +114,10 @@ private[sharing] case class Protocol(minReaderVersion: Int) extends Action {
   override def wrap: SingleAction = SingleAction(protocol = this)
 }
 
+private[sharing] case class NextPageToken(token: String) extends Action {
```

Comment:
Does this need to be synced with the trailer/endStreamToken?

Reply:
+1, but maybe in a separate PR.